frontend/package.json

```diff
@@ -8,28 +8,37 @@
     "dev-server": "webpack-dev-server"
   },
   "dependencies": {
-    "autoprefixer": "^9.4.9",
-    "babel-cli": "6.24.1",
-    "babel-core": "6.25.0",
-    "babel-loader": "7.1.1",
-    "babel-plugin-transform-class-properties": "6.24.1",
-    "babel-plugin-transform-object-rest-spread": "6.23.0",
-    "babel-preset-env": "1.5.2",
-    "babel-preset-react": "6.24.1",
+    "@babel/polyfill": "^7.4.4",
+    "autoprefixer": "^9.6.0",
     "bootstrap": "^4.3.1",
-    "css-loader": "^2.1.0",
-    "live-server": "^1.2.0",
+    "core-util-is": "1.0.2",
+    "jquery": "^3.4.1",
+    "react": "^16.8.6",
+    "react-bootstrap": "^1.0.0-beta.9",
+    "react-dom": "^16.8.6",
+    "react-router-dom": "5.0.1",
+    "react-scripts": "3.0.1",
+    "validator": "8.0.0",
+    "webpack": "4.34.0",
+    "webpack-cli": "3.3.4"
+  },
+  "optionalDependencies": {
+    "fsevents": "^2.0.7"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.4.4",
```

Comment:
Why did you move these babel packages to devDependencies?

Reply:
They aren't used or needed for production code to run, so they are development dependencies.  It helps keep production code cleaner and smaller.

Reply:
Babel complies react code to es5. its need in production for react to work.

Reply:
> Babel complies react code to es5. its need in production for react to work.

No, it doesn't.

https://stackoverflow.com/questions/40143357/do-you-put-babel-and-webpack-in-devdependencies-or-dependencies